controllers/grid/issues/form/IssueForm.php

```diff
@@ -278,7 +277,7 @@ public function execute(...$functionArgs)
 
         $issue->setCoverImageAltText($this->getData('coverImageAltText'), $locale);
 
-        Hook::call('issueform::execute', [$this, $issue]);
+        parent::execute(...$functionArgs);
```

Comment:
Instead of having 
`$this->issue =& $issue;` 
I would suggest to have the following here:
```
$functionArgs[] = $issue;
parent::execute(...$functionArgs);
```
This way, the user of the hook will have both information, about the old and about the new issue -- if, for example, also an access to the issue's old metadata is needed for a reason.

I think that we are not consistent in the forms, but I think the old hook 
`Hook::call('issueform::execute', [$this, $issue]);`
was enabling/considering this (case/example mentioned above), so lets keep it.

Reply:
Ah, yes, I was wrong -- it seems we did not have the old issue metadata...
Thus I think your solution is good, just do we need that call by reference $this->issue =**&** $issue; ?

Reply:
I guess don't need, I will check